jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt

```diff
@@ -771,6 +773,9 @@ class Endpoint @JvmOverloads constructor(
             } else {
                 logger.info("Ignoring empty DtlsFingerprint extension: ${transportInfo.toXML()}")
             }
+            if (CryptexConfig.endpoint) {
+                cryptex = cryptex || fingerprintExtension.cryptex
```

Comment:
I think this should be `&&` now? Otherwise it's just equivalent to the config value. The semantics of `Endpoint.cryptex` is "whether we should use cryptex when encrypting", right?